sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/EmptyNullableValueAccess.cs

```diff
@@ -116,6 +116,35 @@ public void TestComplexCondition(int? i3, double? d3, float? f3)
                 Console.WriteLine(f3.Value); // TODO: Should be NC
             }
         }
+
+        public int CSharp8_SwitchExpressions(bool zero)
+        {
+            int? i = zero switch { true => 0, _ => null };
```

Comment:
We can add one more case with noncompliant access inside switch. What do you think?